helix-query.yaml

```diff
@@ -1,23 +1,22 @@
 indices:
   partsasist:
-    include:
-      - '/**'
-    exclude:
-      - 'nav'
-      - 'footer'
-    target: /query-index.json
-    properties:
-      lastModified:
-        select: none
-        value: parseTimestamp(headers["last-modified"], "ddd, DD MMM YYYY hh:mm:ss GMT")
-      title:
-        select: head > meta[property="og:title"]
-        value: attribute(el, "content")
-      image:
-        select: head > meta[property="og:image"]
-        value: match(attribute(el, "content"), "https:\/\/[^/]+(/.*)")
-      description:
-        select: head > meta[name="description"]
-        value: attribute(el, "content")
-
-
+    # default:
```

Comment:
I had to comment the default keyword because if not, the `hlx up` doesn't work

Reply:
@Buuhuu do you have any insight into why this is?